frame/multisig/src/lib.rs

```diff
@@ -639,15 +603,6 @@ impl<T: Config> Pallet<T> {
 			// Just start the operation by recording it in storage.
 			let deposit = T::DepositBase::get() + T::DepositFactor::get() * threshold.into();
 
-			// Store the call if desired.
-			let stored = if let Some(data) = maybe_call.filter(|_| store) {
-				Self::store_call_and_reserve(who.clone(), &call_hash, data, deposit)?;
-				true
-			} else {
-				T::Currency::reserve(&who, deposit)?;
```

Comment:
@kianenigma is the right? I'm not sure why the reserve has been removed as it has nothing to do with the storage removal.

Reply:
@gavofyork could you shine some light on this please? I'm in the middle of redoing tests and benches and this is crucial to removing some of the redundant tests. If this was a mistake - I can re-introduce the reserve.

Reply:
I am assuming this is a bug, I will reintroduce the deposit in another PR along with fixed tests